front/pages/api/w/[wId]/files/[fileId].ts

```diff
@@ -111,7 +111,7 @@ async function handler(
       }
 
       // Redirect to a signed URL.
-      const url = await file.getSignedUrlForDownload(auth, "original");
+      const url = await file.getUrlSignedForDownload(auth, "original");
```

Comment:
`getUrlSignedForDownload` is less english correct than `getSignedUrlForDownload`.

Reply:
Oops sorry mishap, will fix